src/main/java/TaskList.java

```diff
@@ -0,0 +1,34 @@
+import java.util.ArrayList;
+public class TaskList {
+    ArrayList<Task> tasklist = new ArrayList<Task>();
+
+    public void listTasks() {
```

Comment:
This is a use of "magical number", you can replace with meaning variable like taskCount